src/Microsoft.Health.Dicom.Core/Extensions/DicomTagExtensions.cs

```diff
@@ -24,5 +24,35 @@ public static string GetPath(this DicomTag dicomTag)
             EnsureArg.IsNotNull(dicomTag, nameof(dicomTag));
             return dicomTag.Group.ToString("X4") + dicomTag.Element.ToString("X4");
         }
+
+        /// <summary>
+        /// Get default VR for dicom tag.
+        /// </summary>
+        /// <param name="dicomTag">The dicm tag</param>
+        /// <returns>The default VR.</returns>
+        public static DicomVR GetDefaultVR(this DicomTag dicomTag)
+        {
+            EnsureArg.IsNotNull(dicomTag, nameof(dicomTag));
+            if (dicomTag.IsPrivate)
+            {
+                // Private tag don't have default VR.
+                return null;
+            }
+
+            if (dicomTag.DictionaryEntry == DicomDictionary.UnknownTag)
+            {
+                // this tag is invalid
+                return null;
+            }
+
+            DicomVR[] vrs = dicomTag.DictionaryEntry.ValueRepresentations;
+            if (vrs.Length == 0)
+            {
+                return null;
+            }
+
+            // Use the first valid VR as default VR
+            return vrs[0];
```

Comment:
Would it be possible to just use `return dicomTag.DictionaryEntry.ValueRepresentations.FirstOrdDefault()` here?